pkg/memory-monitor/Makefile

```diff
@@ -32,7 +32,15 @@ $(BIN_DIR)/pressure: $(PRESSURE_OBJ)
 	$(CC) $(CFLAGS) -o $@ $^
 pressure: $(BIN_DIR)/pressure
 
-TARGETS=memory-monitor pressure
+STATISTICS_COLLECTOR_DIR=$(SRC_DIR)/statistics-collector
+STATISTICS_COLLECTOR_SRC=$(wildcard $(STATISTICS_COLLECTOR_DIR)/*.c)
+STATISTICS_COLLECTOR_OBJ=$(patsubst $(SRC_DIR)/%.c,$(OBJ_DIR)/%.o,$(STATISTICS_COLLECTOR_SRC))
+$(BIN_DIR)/statistics-collector: $(STATISTICS_COLLECTOR_OBJ)
+	@mkdir -p $(BIN_DIR)
+	$(CC) $(CFLAGS) -o $@ $^
+statistics-collector: $(BIN_DIR)/statistics-collector
```

Comment:
How will it be cross-compiled?
1. from amd64 to arm64
2. from glibc to musl
3. from OS X to Linux